beater/handlers.go

```diff
@@ -36,11 +36,11 @@ import (
 	conf "github.com/elastic/apm-server/config"
 	"github.com/elastic/apm-server/decoder"
 	"github.com/elastic/apm-server/processor"
-	perr "github.com/elastic/apm-server/processor/error"
-	"github.com/elastic/apm-server/processor/healthcheck"
+	err "github.com/elastic/apm-server/processor/error"
```

Comment:
this will end up shadowed a bunch, some other alias would be better

Reply:
good point